legacy/src/app/partials/node-details.html

```diff
@@ -2277,7 +2277,7 @@ <h2 class="p-heading--four">Create bond</h2>
 
                         <button class="p-button--neutral" data-ng-click="toggleInterfaces()" data-ng-if="showCreateEditButton()">
                             <span data-ng-if="isShowingInterfaces">Hide interfaces</span>
-                            <span data-ng-if="!isShowingInterfaces" data-ng-click="Network table', Edit bond')">Edit bond members</span>
```

Comment:
We should probably preserve this GA handler, so I guess it should be:

```data-ng-click="sendAnalyticsEvent('Network table', 'Edit bond')"```

Reply:
Actually, let's keep this consistently *wrong*, until we do our total GA fixup, so this should be 

```sendAnalyticsEvent('Machine details', 'Edit bond', 'Network tab')```